debug_toolbar/panels/logging.py

```diff
@@ -15,13 +16,33 @@
 
 class LogCollector(ThreadCollector):
 
+    def __init__(self):
+        ThreadCollector.__init__(self)
```

Comment:
Is there a particular reason for using `ThreadCollecter` instead of `super()`? If so, I think it should be documented.